Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -8127,6 +8137,26 @@ public void Actions_CanLookUpActionInAssetByName()
         }, Is.Not.AllocatingGCMemory());
     }
 
+    // https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-1306
```

Comment:
Alternatively, links could be put into a Description attribute:

[Description("See https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-1306")]

Reply:
This is a fair change to this test since specific to reported use-case

Reply:
Addressed in https://github.com/Unity-Technologies/InputSystem/pull/2160/commits/b78188bc492ef186ddfc4a9ec4d6e81cb547d813